spring-cloud-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/NacosConfigProperties.java

```diff
@@ -81,6 +85,26 @@
 	 */
 	private int timeout = 3000;
 
+	/**
+	* nacos maximum number of tolerable server reconnection errors.
+	*/
+	private String maxRetry;
+
+	/**
+	* nacos get config long poll timeout.
+	*/
+	private String configLongPollTimeout;
+
+	/**
+	* nacos get config failure retry time.
+	*/
+	private String configRetryTime;
+
+	/**
+	* nacos enable add a listener when the configuration is obtained.
```

Comment:
This switch is the purpose of the operation to add the listener at the same time to take the initiative to pull the latest configuration to avoid repetition and notice